packages/mgt-components/src/components/mgt-tasks/strings.ts

```diff
@@ -11,5 +11,6 @@ export const strings = {
   addTaskButtonSubtitle: 'Add',
   due: 'Due ',
   addTaskDate: 'Add the task date',
-  assign: 'Assign'
+  assign: 'Assign',
+  dotOptionsTitle: 'More options'
```

Comment:
is this entry used now that it's provided directly in the dot-options component?

Reply:
Nope, removed.